File: 이것이 취업을 위한 코딩 테스트다/구현/시각/시각.cpp
#include <iostream>

using namespace std;

int main() {
  int n, answer;
  cin >> n;

  for (int hour = 0; hour <= n; hour++) {
    for (int min = 0; min <= 59; min++) {
      for (int sec = 0; sec <= 59; sec++) {
        if (hour % 10 == 3 || min / 10 == 3 || min % 10 == 3 || sec / 10 == 3 || sec % 10 == 3) {
          answer++;
        }
      }
    }
  }

  cout << answer;

  return 0;
}

File: 이것이 취업을 위한 코딩 테스트다/구현/시각/시각.java
package 구현.시각;

import java.util.Scanner;

public class 시각 {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int n = sc.nextInt();
    int answer = 0;
  
    for(int hour = 0; hour <= n; hour++) {
      for(int min = 0; min <= 59; min++) {
        for(int sec = 0; sec <= 59; sec++) {
          if (hour % 10 == 3 || min / 10 == 3 || min % 10 == 3 || sec / 10 == 3 || sec % 10 == 3) {
            answer++;
          }
        }
      }
    }
    System.out.println(answer);
  }
}
